requirements.txt

```diff
@@ -1,7 +1,9 @@
-jsonschema>=2.5.1
-flask>=0.10.1
-PyYAML>=3.11
-requests>=2.9.1
-six>=1.7
-strict-rfc3339>=0.6
-swagger_spec_validator>=2.0.2
+jsonschema>=2.5.1,<3
+flask>=0.10.1,<0.12
+PyYAML>=3.11,<4
+requests>=2.9.1,<3
```

Comment:
I admit that I haven't, though it is a very common convention and quite likely to be the case - especially for the large and well-known projects that `connexion` depends on.  

If `connexion` does have dependencies that are willing to break APIs without signaling this in the release number, then there isn't a lot that can be done to protect ourselves from that.

However let's not make the perfect the enemy of the good...